.github/workflows/test.yml

```diff
@@ -57,3 +57,7 @@ jobs:
 
     - name: Install pandas-stubs and run tests on the installed stubs
       run: poetry run poe test_dist
+
+    - if: matrix.python-version == '3.10' && matrix.os == 'ubuntu-latest'
```

Comment:
probably worth adding a comment here that says that this test has to be last, otherwise any additional test below it will use the nightly build

Reply:
Could probably run it on all platforms for 3.10.